samples/routing/fair-distribution/Core_6/Client/Program.cs

```diff
@@ -83,9 +83,9 @@ static void AddRouting(EndpointConfiguration endpointConfiguration)
         #region Routing
 
         const string server = "Samples.FairDistribution.Server";
-        var routing = endpointConfiguration.Routing();
+        var routing = endpointConfiguration.UseTransport<MsmqTransport>().Routing();
         routing.RouteToEndpoint(typeof(PlaceOrder), server);
-        routing.Mapping.Physical.Add(
+        routing.GetSettings().GetOrCreate<EndpointInstances>().Add(
```

Comment:
Agree with bringing in instance mapping file.

The key here is the distinction between logical mapping to endpoints, which should be accomplished in code by a developer, vs. the physical mapping of logical endpoints to physical machines, which is a deployment time concern and should always be malleable by an operations person. We're very opinionated that you shouldn't be hardcoding that, which means it should not be available/discoverable starting from EndpointConfiguration.

Just putting it behind an `.Advanced.` in the API is just putting the gun on a slightly taller shelf.

So availability through a feature is a way to provide it to a (different use of this word) customized routing feature that would need it. We don't intend for customers to find this themselves. They would see it in a sample or ask us directly. This seems preferable to yet another abstraction along the lines of `IControlLogicalToPhysicalRouting` or something like that.